### ISSUE TITLE
can support IDA 7.X Python3?

### ISSUE THREAD

**Issue by cryzlasm** [fireeye/flare-emu#36]
require  unicorn==1.0.1  

can't install in new IDA version, python3

**Comment by jtbennett-fe** [comment]
Sorry about that, the requirements.txt file was left in the repo even though it is not in use. Please follow these simple installation instructions instead: https://github.com/fireeye/flare-emu#installation

**Comment by cryzlasm** [comment]
thanks, it work

**Comment by jtbennett-fe** [comment]
you're welcome, thank you for bringing this to our attention!

**Comment by wanitaqing** [comment]
Hi & sorry to hop into this issue! May I ask how did you (@cryzlasm) get it to work? I've following the [installation guide](https://github.com/fireeye/flare-emu#installation) & also installed the unicorn python package using `pip install unicorn`.

![image](https://user-images.githubusercontent.com/47132451/116198743-ed069100-a768-11eb-9215-6ed01dcf4aa9.png)
_Python files in python folder of IDA_

![image](https://user-images.githubusercontent.com/47132451/116198861-0f98aa00-a769-11eb-93d0-21dce446e77e.png)
_Unicorn folders in site-packages folder_

**Comment by wanitaqing** [comment]
Hi! I apologise for hopping onto this thread, but may I ask how did you (@cryzlasm) managed to install it? I've followed the [installation guide](https://github.com/fireeye/flare-emu#installation) and also installed the unicorn module using `pip install unicorn`

![image](https://user-images.githubusercontent.com/47132451/116201654-4de39880-a76c-11eb-8a05-e631974a1ca8.png)
_Unicorn folders in Python3.8 folder. I also checked and the unicorn.dll file is in this folder "unicorn/lib"_

![image](https://user-images.githubusercontent.com/47132451/116201860-8b482600-a76c-11eb-81a8-ac33801a890b.png)
_the 3 python files need for flare-emu_
